target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh

```diff
@@ -162,6 +162,27 @@ platform_do_upgrade() {
 		CI_ROOTPART="ubi_rootfs"
                 nand_do_upgrade "$1"
                 ;;
+	unielec,u7981-01*)
+		local rootdev="$(cmdline_get_var root)"
+		rootdev="${rootdev##*/}"
+		rootdev="${rootdev%p[0-9]*}"
+		case "$rootdev" in
+		mmc*)
+			CI_ROOTDEV="$rootdev"
+			CI_KERNPART="kernel"
+			CI_ROOTPART="rootfs"
+			emmc_do_upgrade "$1"
+			;;
+		mtdblock*)
```

Comment:
Is there are version with NOR flash as well?
I see two board variants added, but three sysupgrade variants, doesn't match...